src/components/SignIn.js

```diff
@@ -0,0 +1,20 @@
+
+const SignIn = ({ setSign }) => {
+
+
+
+
+    return (
+        <div className="signin-form">
+            <form className="signIn">
```

Comment:
I would recommend checking  [formik](https://formik.org/) to handle your forms for you and also it has built-in error messages